python/data/ArtusConfigs/Run2MSSM/Includes/settingsJEC.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+import Artus.Utility.logger as logger
+log = logging.getLogger(__name__)
+
+import re
+import json
+import copy
+import Artus.Utility.jsonTools as jsonTools
+import Kappa.Skimming.datasetsHelperTwopz as datasetsHelperTwopz
+import importlib
+
+def build_config(nickname):
+  config = jsonTools.JsonDict()
+  datasetsHelper = datasetsHelperTwopz.datasetsHelperTwopz("Kappa/Skimming/data/datasets.json")
+  
+  
+  ## fill config:
+  # includes
+  includes = [
+    ]
+  for include_file in includes:
+    analysis_config_module = importlib.import_module(include_file)
+    config += analysis_config_module.build_config(nickname)
+  
+  # explicit configuration
+  if re.search("Run2015", nickname):
+    config["JetEnergyCorrectionParameters"] = [
+      "#$CMSSW_BASE/src/HiggsAnalysis/KITHiggsToTauTau/data/jec/Fall15/Fall15_25nsV2_DATA_L1FastJet_AK4PFchs.txt",
```

Comment:
what is the # needed for?

Reply:
just an import error from the old json-config. It's commented out now